<launch>
  <remap from="/tf" to="/ocs/tf" />
  <remap from="/joint_states" to="/ocs/joint_states" />
  <remap from="/robot_description" to="/ocs/robot_description" />

  <node pkg="drc_task_common" type="rviz_status.py" name="rviz_status">
  </node>
  <!-- robot motion -->
  <include file="$(find drc_task_common)/launch/robot_motion.launch" >
    <arg name="default_frame_id" default="$(arg default_frame_id)" />
    <arg name="ik_server_launch" default="$(arg ik_server_launch)" />
  </include>

  <!-- object model -->
  <include file="$(find drc_task_common)/launch/transformable_model.launch" >
    <arg name="default_frame_id" default="$(arg default_frame_id)" />
  </include>

  <!-- robot model -->
  <include file="$(find drc_task_common)/launch/robot_model.launch" >
    <arg name="default_frame_id" default="$(arg default_frame_id)" />
    <arg name="robot_model_config" default="$(arg robot_model_config)" />
  </include>

  <!-- teleop device -->
  <include file="$(find drc_task_common)/launch/teleop_device.launch" >
    <arg name="default_frame_id" default="$(arg default_frame_id)" />
    <arg name="enable_trackball_head" default="false" />
  </include>

  <!-- teleop interface -->
  <include file="$(find drc_task_common)/launch/teleop_interface.launch" />

  <!-- rviz -->
  <node pkg="rviz" type="rviz" name="operator_rviz" args="-d $(find drc_task_common)/config/drc_task_common.rviz"
        respawn="true"/>
</launch>
